### ISSUE TITLE
Error: Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: Server error: `ERROR 42000 (1064): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1' 

### ISSUE THREAD

**Issue by divyenduz** [prisma/migrate#546]
Command: `prisma migrate up --experimental`
Version: `2.3.0`
Binary Version: `e11114fa1ea826f9e7b4fa1ced34e78892fe8e0e`
Report: https://prisma-errors.netlify.app/report/9754 
OS: `x64 win32 10.0.19041`

JS Stacktrace:
```
Error: Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: Server error: `ERROR 42000 (1064): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1'
   0: migration_core::api::ApplyMigration
           with migration_id="20200804163056-init"
             at migration-engine\core\src\api.rs:77)
    at Object.<anonymous> (C:\workspace\holdem-server\node_modules\@prisma\cli\build\index.js:2:2135004)
    at MigrateEngine.handleResponse (C:\workspace\holdem-server\node_modules\@prisma\cli\build\index.js:2:2133212)
    at LineStream.<anonymous> (C:\workspace\holdem-server\node_modules\@prisma\cli\build\index.js:2:2134669)
    at LineStream.emit (events.js:310:20)
    at addChunk (_stream_readable.js:286:12)
    at readableAddChunk (_stream_readable.js:268:9)
    at LineStream.Readable.push (_stream_readable.js:209:10)
    at LineStream.Transform.push (_stream_transform.js:152:32)
    at LineStream._pushBuffer (C:\workspace\holdem-server\node_modules\@prisma\cli\build\index.js:2:1819384)
    at LineStream._transform (C:\workspace\holdem-server\node_modules\@prisma\cli\build\index.js:2:1819205)
```

Rust Stacktrace:
```
Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: Server error: `ERROR 42000 (1064): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')' at line 1'
   0: migration_core::api::ApplyMigration
           with migration_id="20200804163056-init"
             at migration-engine\core\src\api.rs:77)
```

**Comment by albertoperdomo** [comment]
@tomhoule will attempt to repro

**Comment by tomhoule** [comment]
I investigated this, and it is a schema parser issue: enums with no values/variants are currently valid, but they can't be created as such in the database.

Issue: https://github.com/prisma/prisma-engines/issues/1070

**Comment by albertoperdomo** [comment]
We'll attempt to repro once it's been addressed in the schema parser.

**Comment by tomhoule** [comment]
Open PR to follow progress on this: https://github.com/prisma/prisma-engines/pull/1093

**Comment by albertoperdomo** [comment]
Fixed via prisma/prisma-engines#1093
